### ISSUE TITLE
v0.3.1: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by yheuhtozr** [umaranis/svelte-lexical#43]
Hi, I'm trying out this module and the latest version seems to crash with the following error.
The component is rendered with no errors using `svelte-lexical@0.3.0`, so I think it has something to do with `0.3.1`.

```
$ yarn run dev

...

[vite] Error when evaluating SSR module /node_modules/svelte-lexical/src/core/plugins/collaboration/collaboration.ts: failed to import "y-websocket"

...

Internal server error: Cannot use import statement outside a module
      at internalCompileFunction (node:internal/vm:73:18)
      at wrapSafe (node:internal/modules/cjs/loader:1176:20)
      at Module._compile (node:internal/modules/cjs/loader:1218:27)
      at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
      at Module.load (node:internal/modules/cjs/loader:1117:32)
      at Module._load (node:internal/modules/cjs/loader:958:12)
      at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
      at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
/*****/node_modules/y-websocket/src/y-websocket.js:7
import * as Y from 'yjs' // eslint-disable-line
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1176:20)
    at Module._compile (node:internal/modules/cjs/loader:1218:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
```

**Comment by umaranis** [comment]
Hi @yheuhtozr ,
I am not facing this issue in `playground` and `sveltekit` demo projects. Probably the issue caused by a combination of `sveltekit` and `CollaborationPlugin`. 

Could you confirm if you are using `CollaborationPlugin` in a sveltekit project?

**Comment by yheuhtozr** [comment]
Thank you. I'm using SvelteKit but I don't think I explicitly turns on `CollaborationPlugin`. My current import is:
``` ts
import {
	Composer,
	ContentEditable,
	PlainTextPlugin,
	HistoryPlugin,
	PlaceHolder
} from 'svelte-lexical';
```
I don't have a plan to use `CollaborationPlugin` anyway, so it'd be fine as well if there is a way to turn it off manually.

**Comment by umaranis** [comment]
Hi @yheuhtozr ,
Seems like the issue is with JS module import. Do you have `"type": "module"` in your `package.json` file?

**Comment by yheuhtozr** [comment]
Yes I have. I think it came by default. Does it cause problem?

![image](https://user-images.githubusercontent.com/84892012/234004075-def85fa6-fc48-4b13-bae8-191e35e7075f.png)

**Comment by umaranis** [comment]
Hi @yheuhtozr ,
I updated all the packages to the latest version. Now, I am able to reproduce the issue. It means that some change in one of the dependencies probably `vite` is causing it.

It only happens in `dev`, release build is not impacted which further suggests that the issue is related to some change in behaviour in `vite`.

**Comment by umaranis** [comment]
Found the root cause... `y-websocket` package doesn't have `"type": "module"` in `package.json`. Vite doesn't like that with dev build.  

For now, I have moved out `y-websocket` code into playground instead of the core library to resolve the issue.

**Comment by umaranis** [comment]
Hi @yheuhtozr ,

I have fixed the issue and released a new package, version 0.3.2. 

Could you please try with 0.3.2 and confirm?

**Comment by yheuhtozr** [comment]
@umaranis Now it just works fine! Thank you for the quick update.
